### ISSUE TITLE
Having difficulty on reproducing MATH (AIME / AMC) results

### ISSUE THREAD

**Issue by spacegoing** [Unakar/Logic-RL#66]
Hi Unakar,

I successfully trained model with bsz=128, ppo mini bsz=32, rollout n =16, lr=4e-6. And converging curve as follows:

<img width="635" alt="Image" src="https://github.com/user-attachments/assets/6bc7b39f-4f16-4637-9dc9-7f4915871314" />

However, when try to eval ckpts' AIME / AMC accuracy, I see no siginificant improvement for all ckpts in Math, even degragaded performance:

<img width="596" alt="Image" src="https://github.com/user-attachments/assets/8aaabccc-0e33-46aa-a8ac-90d50aa86120" />

Would u pls share training script u used for Section "RQ 4: Can the Model Generalize to Out-of-Distribution (OOD) Tasks?"

Many thanks!

**Comment by Unakar** [comment]
my script use train_bsz=8,rollout=8, lr=4e-7 (reinforce++)

**Comment by mohamad-amin** [comment]
@Unakar Could you please the other hyperparameters used as well? Specifically, `ppo_mini_batch_size` and `ppo_micro_batch_size`. 
If you could share the script that you used for achieving the reported results that would be much appreciated as well!
